server/routes/api/__test__/auth.unit.test.js

```diff
@@ -0,0 +1,56 @@
+const request = require('supertest');
+const app = require('../../../../app');
+const User = require('../../../models/user');
+
+describe('POST Auth/Login', () => {
+  const emailForTesting = 'testUser@test.com';
+
+  beforeAll(() => {
+    return User.create({ email: emailForTesting, password: 'test' });
+  });
+
+  afterAll(() => {
+    return User.findOneAndDelete({ email: emailForTesting });
+  });
+
+  it('will return error with invalid email', async () => {
+    const res = await request(app)
+      .post('/api/auth/login')
+      .type('form')
+      .send('email=test@test'); // send post with invalid invalid email
```

Comment:
Same recommendation as your other testing PR. Let's convert this to an object inside of the send.

Reply:
Done.